src/components/CertificateDropZone/CertificateDropZone.js

```diff
@@ -98,7 +100,7 @@ const CertificateDropzone = ({
     onDrop={acceptedFiles =>
       onFileDrop(acceptedFiles, handleCertificateChange, handleFileError)
     }
-    className="h-100"
+    className={css.dropzone}
```

Comment:
I think no need to change the class. bootsytrap "h-100" class is working fine.

Reply:
not on my computer

![image](https://user-images.githubusercontent.com/503185/66461608-fe674a00-eaab-11e9-8372-31e5bf8f3266.png)

Reply:
![image](https://user-images.githubusercontent.com/503185/66461653-150da100-eaac-11e9-934f-7cc397b49b89.png)